frontend/express/public/javascripts/countly/countly.vue.components.js

```diff
@@ -372,4 +373,2886 @@
+                        this.navigatedIndex = this.visibleItems.length - 1;
+                    }
+                    else {
+                        this.navigatedIndex = this.getNextNonGroupIndex(this.navigatedIndex - 1, -1);
+                    }
+
+                    this.scrollToNavigatedIndex();
+                },
+                downKeyEvent: function() {
+                    if (!this.isKeyboardNavAvailable) {
+                        return;
+                    }
+
+                    if (this.navigatedIndex === null) {
+                        this.navigatedIndex = 0;
+                    }
+                    else {
+                        this.navigatedIndex = this.getNextNonGroupIndex(this.navigatedIndex + 1, 1);
+                    }
+
+                    this.scrollToNavigatedIndex();
+                },
+                escKeyEvent: function() {
+                    if (this.navigatedIndex === null && this.opened) {
+                        this.close();
+                        return;
+                    }
+                    else if (this.navigatedIndex !== null) {
+                        this.navigatedIndex = null;
+                    }
+                },
+                enterKeyEvent: function() {
+                    if (this.navigatedIndex === null) {
+                        return;
+                    }
+
+                    this.selectNavigatedElement();
+                }
+            },
+            watch: {
+                value: {
+                    immediate: true,
+                    handler: function(passedValue) {
+                        if (typeof passedValue !== 'object') {
+                            var item = this.findItemByValue(passedValue);
+                            if (item) {
+                                this.setItem(item);
+                            }
+                            else {
+                                this.setItem({name: passedValue, value: passedValue});
+                            }
+                        }
+                    }
+                },
+                opened: function(newValue) {
+                    if (!newValue) {
+                        this.tempSearchQuery = "";
+                        this.searchQuery = "";
+                        this.navigatedIndex = null;
+                    }
+                },
+                tempSearchQuery: _.debounce(function(newVal) {
+                    this.searchQuery = newVal;
+                }, 500),
+                searchQuery: function() {
+                    this.fireDynamicSearch();
+                },
+                items: {
+                    immediate: true,
+                    handler: function() {
+                        this.waitingItems = false;
+                    }
+                },
+                visibleItems: function() {
+                    this.navigatedIndex = null; // reset navigation on visible items change
+                }
+            },
```

Comment:
Codacy found an issue: [Multiline support is limited to browsers supporting ES5 only.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6339769)